### ISSUE TITLE
Sample problem in boundary-distribution definition: bounds are not legal.

### ISSUE THREAD

**Issue by MatthiVH** [SALib/SALib#329]
The saltelli sampler can handle other distributions in the problem definition, however the latin hypercube sampler can not.

Example-code:
```python
problem = { 'num_vars': 3,
                    'names': ['P1', 'P2', 'P3'],
                    'bounds': [[0.9, 1.1], [5, 1], [3.4, 0.1]],
                    'dists': ['unif', 'norm', 'lognorm']}

#generating samples
saltelli_samples = saltelli.sample(prob_dists_code, 1000, calc_second_order=True) -> works fine

#generating samples
latin_samples = latin.sample(prob_dists_code, 1000) -> ValueError: Bounds are not legal.
```

I assume the latin sampler doesn't recognize the 'dists'-statement in the problem-definition and assumes all uniform.
Can this be fixed?

Regards,
Matthias

**Comment by willu47** [comment]
Hi @MatthiVH - thanks for the bug report. We'll investigate. It should be a relatively easy fix (I hope).

**Comment by MatthiVH** [comment]
Thanks, that's really helpful.

**Comment by ConnectedSystems** [comment]
Addressed by PR for v1.4 so I will close this.

Thanks again @lbteixeira
